klippy/toolhead.py

```diff
@@ -290,7 +290,7 @@ def __init__(self, config):
                                             self._handle_shutdown)
         # Load some default modules
         modules = ["gcode_move", "homing", "idle_timeout", "statistics",
-                   "manual_probe", "tuning_tower"]
+                   "manual_probe", "tuning_tower", "garbage_collection"]
```

Comment:
It was mentioned to look at how toolhead auto loads modules. I'm not certain that pointer was intended to say we should do the autoload here in toolhead, or if there would be a more preferred place to do the `load_object`?